runtime/etc/Makefile.mli-static

```diff
@@ -0,0 +1,81 @@
+# Copyright 2004-2019 Cray Inc.
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Makefile.mli-static
+#
+
+#
+# TODO
+#
+# There's a slightly more complicated build process going on here, because we
+# have 3 deliverables (for now):
+#
+#   - A client facing library file (static or shared)
+#   - A multi-locale launcher
+#   - A multi-locale executable
+#
+# We build the server as if it's a multi-locale executable, and then build the
+# client as if it's a (static, for this file) library.
+#
+
+include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.include
+
+#
+# This is common code included wholesale in each MLI compile (for now).
+#
+CHPL_MLI_INCLUDES = $(CHPL_MAKE_HOME)/runtime/etc/mli
+
+#
+# You MUST SET THIS or compilation will fail!
+#
+ifndef CHPL_ZMQ_LOC
```

Comment:
Is this a new environment variable requirement?  If so, it should be documented in the PR message (and in the multilocale library documentation we eventually generate - maybe track that via an issue?)

Reply:
Ok.

Reply:
This does not appear to have been added to the PR message, and I don't recall seeing an issue opened to track it yet.